docker/rust/release/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM stablex-binary-private
+ARG BINARY_BASE=stablex-binary-public
+
+FROM ${BINARY_BASE}
```

Comment:
Given this file is so small, maybe it makes sense to have two of them (public/private) and also export some env vars (e.g. SOLVER_TYPE should be set to open_solver in the public image).

Reply:
>SOLVER_TYPE should be set to open_solver in the public image

I could do that. But wouldn't it feel strange, that we are setting the solvertype for the one image,  but not for the other image? ( We can't know whether we have the standard-solver or fallback-solver?) Or should we always assume the standard-solver and overwrite this variable just for the fallback solver?

Reply:
I think that would be good. The less manual configuration one has to do the easier it will be to run this for external people.